src/domain/application/AppConfig.ts

```diff
@@ -0,0 +1,94 @@
+class AppConfig {
+  static get apiUrl() {
+    return getEnvOrError(process.env.REACT_APP_API_URI, 'REACT_APP_API_URI');
+  }
+
+  static get oidcAuthority() {
+    const origin = getEnvOrError(
+      process.env.REACT_APP_OIDC_AUTHORITY,
+      'REACT_APP_OIDC_AUTHORITY'
+    );
+    return new URL(origin).href;
+  }
+
+  /**
+   * The audiences used in the OIDC.
+   * 
+   * @example
+   * // In Tunnistamo it can be left undefined.
+   * ["https://api.hel.fi/auth/kukkuu"]
```

Comment:
This comment is slightly confusing, what is meant here? What happens if the audiences is left undefined for Tunnistamo? Is it not used at all? Does it use a default value in that case? Is the list given under the Tunnistamo comment—which doesn't end in `:`—meant as an example working value for Tunnistamo? Not a big deal, but could use more clarity IMHO.